src/main/java/org/fundacionjala/coding/daniel/Kata2.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.daniel;
+
+/**
+ * Created by Administrator on 30/08/17.
+ */
+public class Kata2 {
+
+    /**
+     * Kata2.
+     */
+    protected Kata2() {
```

Comment:
Remove this unnecessary constructor.

Reply:
otherwise you will need to test it as private.
But in this case just remove it.
and remove the static reserved word in the method solution.